docs/src/pages/components/slider-styled/ContinuousSlider.js

```diff
@@ -37,6 +37,15 @@ export default function ContinuousSlider() {
         Disabled slider
       </Typography>
       <Slider disabled defaultValue={30} aria-labelledby="disabled-slider" />
+      <Slider
```

Comment:
The changes in the examples will be reverted before merging. They just illustrate how the prop can be used.